micrometer-observation/src/main/java/io/micrometer/observation/ObservationView.java

```diff
@@ -25,6 +25,15 @@
  */
 public interface ObservationView {
 
+    /**
+     * Returns the {@link ObservationRegistry} attached to this observation.
+     * @return corresponding observation registry
+     * @since 1.10.10
+     */
+    default ObservationRegistry getObservationRegistry() {
```

Comment:
Are we able to operate on the `ObservationRegistry.ObservationConfig` level instead? All the calls to the registry to obtain the current `Observation`/`Scope` can go to the static, global instance.

Reply:
They cannot because we need an `ObservationRegistry` to create an `Observation`, not an `ObservationConfig`